recipes-extended/baseboxd-tools/baseboxd-tools/basebox-init-l2-learning-config

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Copyright (C) 2024 Rubens Figueiredo of BISDN GmbH
+# Bash script to set up a basic learning L2 switch configuration
+# Creates systemd files in /etc/systemd/network which sets up the
+# switch to provide L2 forwarding.
+
+set -e
+
+# Setup environment variables
+CONFIG_DIR=/etc/systemd/network
+FILE_INFIX="l2-learn"
+USAGE="Create files to set up L2 switch.
+usage: $(basename "$0") [OPTIONS]
+
+[OPTIONS]
+-h show this message
+-w write the files to $CONFIG_DIR
+-d clean up auto-generated files
+-s show files created but don't write anything
+"
+
+cleanup() {
+  rm *.$FILE_INFIX.netdev
+  rm *.$FILE_INFIX.network
+}
+
+# Generate port.network files
+# Call with port_network true/false portname
+# The first argument writes (true) or shows the file (false)
+# The second argument sets the port name
+port_network() {
+  show=$1
+
+  FILE="# Autogenerated by $(basename "$0")
+[Match]
+Name=port*
+
+[Network]
+Bridge=swbridge
+
+[BridgeVLAN]
+PVID=1
+EgressUntagged=1
+VLAN=1"
```

Comment:
This is redundant and not needed since you specify `DefaultPVID=1` for the bridge.